### PR TITLE
fix: update destructive button styles to be filled

### PR DESCRIPTION
**Pull request by SiTaggart**
Update the destructive buttons to be ~outlined~ filled.

also get storybook serving colfax correctly for the SG theme

### CODE REVIEW

File: .storybook/addons/theme-switcher-addon/index.js [hunk 1]
```diff
@@ -26,7 +51,12 @@ class ThemeWrapper extends React.Component {
   };
 
   render() {
-    return <Theme.Provider theme={this.state.theme}>{this.props.children}</Theme.Provider>;
+    return (
+      <Theme.Provider theme={this.state.theme}>
+        <Global styles={globalStyles} />
+        <Box padding="space40">{this.props.children}</Box>
```
Comment (TheSisb) [inline .storybook/addons/theme-switcher-addon/index.js]: Thank you 👍

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "prestart": "yarn prebuild",
     "start": "yarn build:tokens && concurrently \"yarn:start:tokens\" \"yarn:start:storybook\"",
     "start:all": "yarn build:tokens && concurrently \"yarn:start:tokens\" \"yarn:start:storybook\" \"yarn:start:website\"",
-    "start:storybook": "start-storybook",
+    "start:storybook": "start-storybook -s ./packages/paste-website/static -p 9001",
```
Comment (TheSisb) [inline package.json]: Not sure how I feel about this coupling.  Could we move these static assets out of the `paste-website` package?

Comment (SiTaggart) [reply package.json]: Seems like Gatsby is pretty fixed on needing that static directory in the root of the gatsby project so it know where to grab the static assets from. https://www.gatsbyjs.org/docs/static-folder/

I can duplicate them for storybook, but that seems not ideal either.

Comment (TheSisb) [reply package.json]: While I agree it isn't ideal, duplication here might be the better alternative.  As a rule we should try to follow, the only things that can reach into other parts of the monorepo are build tooling and npm packages (lerna).  The duplication helps prevent unintended and unexpected regressions.

Comment (SiTaggart) [reply package.json]: Fair, dumped a copy of it under the `.storybook` directory

**Pull request merged**